nmdc_runtime/site/ops.py

```diff
@@ -1043,6 +1045,48 @@ def site_code_mapping() -> dict:
         )
 
 
+@op(
+    required_resource_keys={"mongo"},
+    config_schema={
+        "source_ontology": str,
+        "output_directory": Field(Noneable(str), default_value=None, is_required=False),
+        "generate_reports": Field(bool, default_value=True, is_required=False),
+    },
+)
+def load_ontology(context: OpExecutionContext):
+    cfg = context.op_config
+    source_ontology = cfg["source_ontology"]
+    output_directory = cfg.get("output_directory")
+    generate_reports = cfg.get("generate_reports", True)
+
+    if output_directory is None:
+        output_directory = os.path.join(os.getcwd(), "ontology_reports")
+
+    # Redirect Python logging to Dagster context
+    handler = logging.Handler()
```

Comment:
I added this so that the output of the ontology load is reflected in the Dagster UI.

Reply:
This is a great TIL! I didn't know this is how we could redirect output to the Dagster console/UI. Will steal this for my work 💕